astropy/modeling/polynomial.py

```diff
@@ -1136,6 +1175,23 @@ class Legendre2D(OrthoPolynomialBase):
         range of the y independent variable
     **params : dict
         keyword: value pairs, representing parameter_name: value
+
+    Notes
+    -----
+    Model formula:
+
+    .. math::
+
+        P(x) = \sum_{i=0}^{i=n}C_{i} * L_{i}(x)
+
+    where :math: `L_{i}` is the corresponding Legendre polynomial.
```

Comment:
I think the travis failure is due to the space between `:math:` and `` `L_{i}` ``